android/src/main/java/com/pdftron/pdftronflutter/helpers/ViewerImpl.java

```diff
@@ -186,6 +187,11 @@ public void onBookmarkModified() {
             }
         }
 
+        @Override
+        public void onBookmarkModified(@NonNull List<UserBookmarkItem> bookmarkItems) {
```

Comment:
move everything in `onBookmarkModified()` to here as `onBookmarkModified()` is getting deprecated